### ISSUE TITLE
[Bug] <Accessibility: Missing alt attributes for images>

### ISSUE THREAD

**Issue by LMDcoded** [creativecommons/index-prototype#71]
## Reproduction
1. 
2. 

## Resolution

- [x] I would be interested in resolving this bug.
